R/check_suggested.R

```diff
@@ -10,27 +10,25 @@
 #' @noRd
 check_suggested <- function(packages,
                             use,
-                            top_level_fun = NULL,
-                            message = NULL,
-                            version = NULL) {
+                            top_level_fun = NULL
+                            ) {
   # If NA, inform that package isn't installed, but only in interactive mode
   if (is.na(use)) {
     use <- is_interactive()
     if (!use) {
       return(FALSE)
     }
 
-    installed <- map_lgl(packages, ~ {
-      installed <- is_installed(., version = version)
-      if (!installed) {
-        if (is.null(message)) {
-          # FIXME: Mention version
-          message <- glue("`{top_level_fun}()` is improved by the '{.}' package. Consider `install.packages(\"{.}\")`.")
-        }
-        inform(message)
-      }
-      installed
+    installed <- map_lgl(packages, function(pkg) {
+      is_installed(pkg)
     })
```

Comment:
```suggestion
    installed <- map_lgl(packages, is_installed)
```

Can this move even higher, outside `if (!use)` ? We could return immediately if all packages are installed, in all cases.